frontend/src/pages/SettingsPage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useCheckAuthenticated } from '@semapps/auth-provider';
+import { email, required, useAuthProvider, useNotify, useTranslate, } from 'react-admin';
+import { SimpleForm, TextInput } from 'react-admin';
+import { Container } from '@material-ui/core';
+import HeaderTitle from '../layout/HeaderTitle';
+
+const validateEmail = [required(), email('app.validation.email')]
+const validateConfirmNewPassword = [(value, { newPassword, confirmNewPassword }) => {
+	if (!newPassword) return;
+
+	if (newPassword !== confirmNewPassword) {
+		return 'app.validation.confirmNewPassword'
+	}
+	return;
+}];
+
+
+const SettingsPage = () => {
+	const translate = useTranslate();
+	const notify = useNotify();
+	const { identity } = useCheckAuthenticated();
+	const authProvider = useAuthProvider();
+
+	const [formDefaultValue, setFormDefaultValue] = React.useState({
+		email: "",
+		currentPassword: "",
+		newPassword: "",
+		confirmNewPassword: ""
+	})
+
+	React.useEffect(() => {
+		authProvider.getAccountSettings().then((res) => {
+			setFormDefaultValue({ ...formDefaultValue, email: res.email })
+		});
+		// eslint-disable-next-line
+	}, [setFormDefaultValue, authProvider])
+
+	if (!identity?.id) return null;
+
+	return (
+		<>
+			<HeaderTitle>
+				{translate('app.page.settings')}
+			</HeaderTitle>
+			<Container>
+				<SimpleForm initialValues={formDefaultValue}
```

Comment:
For your information, I usually use `react-final-form` Form component for forms that are not linked with React-Admin, but then I have to create my own "Save" button (see [here](https://github.com/assemblee-virtuelle/semapps/blob/next/src/frontend/packages/interop-components/src/forms/LexiconImportForm.js) for an example). So your solution is fine, even though the code is a bit complicated